src/main/resources/org/sagebionetworks/web/client/widget/entity/annotation/AnnotationEditorViewImpl.ui.xml

```diff
@@ -1,25 +1,41 @@
 <ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
 	xmlns:g="urn:import:com.google.gwt.user.client.ui"
 	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
+	xmlns:b.html="urn:import:org.gwtbootstrap3.client.ui.html"
 	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html"
 	xmlns:t="urn:import:org.sagebionetworks.web.client.view.bootstrap.table">
-	<t:Table>
+	<t:Table addStyleNames="annotations-editor-view" width="100%">
+		<t:TableRow>
+			<t:TableHeader addStyleNames="min-width-200">
+				<b.html:Text>Key</b.html:Text>
```

Comment:
already importing org.gwtbootstrap3.client.ui.html as bh, so this could be rewritten as <bh:Text>